hphp/hack/src/errors/typing_error.ml

```diff
@@ -1166,7 +1167,7 @@ module Primary = struct
     let missing_assign pos =
       (Error_code.MissingAssign, (pos, "Please assign a value"), [], [])
 
-    let non_void_annotation_on_return_void_function is_async pos =
+    let non_void_annotation_on_return_void_function is_async pos hint_pos=
```

Comment:
Don't forget to format your code! You should be able to right click in VS Code and format it, or you can run `ocamlformat` from the command line.

```suggestion
    let non_void_annotation_on_return_void_function is_async pos hint_pos =
```